ckan/templates-bs3/user/edit_user_form.html

```diff
@@ -1,10 +1,10 @@
 {% import 'macros/form.html' as form %}
 
-{% block form %}
+{% block user_edit_form %}
```

Comment:
Do not change existing block names as it might break extensions

Reply:
Reverted it back to `{% block form %}`